src/commons/colors.scss

```diff
@@ -0,0 +1,7 @@
+$background-primary: #f3f3f3;
```

Comment:
**Awesome**: Using variables for theming
_Suggestion_: Keep capitalization consistent for hex values

Reply:
Got it. Changed.